docs/userguide/formatparse/numbers/legacy-numberformat.md

```diff
@@ -0,0 +1,196 @@
+    Returns the percentage format for the current locale or for a specified
+    locale.
+
+4.  **createScientificInstance()**
+    Returns the scientific number format for the current locale or for a
+    specified locale.
+
+To create a format for spelled-out numbers, use a constructor on
+RuleBasedNumberFormat (§).
+
+#### Currency Formatting
+
+Currency formatting, i.e., the formatting of monetary values, combines a number
+with a suitable display symbol or name for a currency. By default, the currency
+is set from the locale data from when the currency format instance is created,
+based on the country code in the locale ID. However, for all but trivial uses,
+this is fragile because countries change currencies over time, and the locale
+data for a particular country may not be available.
+
+For proper currency formatting, both the number and the currency must be
+specified. Aside from achieving reliably correct results, this also allows to
+format monetary values in any currency with the format of any locale, like in
+exchange rate lists. If the locale data does not contain display symbols or
+names for a currency, then the 3-letter ISO code itself is displayed.
+
+The locale ID and the currency code are effectively independent: The locale ID
+defines the general format for the numbers, and whether the currency symbol or
+name is displayed before or after the number, while the currency code selects
+the actual currency with its symbol, name, number of digits, and [rounding
+mode](rounding-modes.md).
+
+In ICU and Java, the currency is specified in the form of a 3-letter ISO 4217
+code. For example, the code "USD" represents the US Dollar and "EUR" represents
+the Euro currency.
+
+In terms of APIs, the currency code is set as an attribute on a number format
+object (on a currency instance), while the number value is passed into each
+format() call or returned from parse() as usual.
+
+1.  ICU4C (C++) NumberFormat.setCurrency() takes a Unicode string (const UChar
+    \*) with the 3-letter code.
+
+2.  ICU4C (C API) allows to set the currency code via unum_setTextAttribute()
+    using the UNUM_CURRENCY_CODE selector.
+
+3.  ICU4J NumberFormat.setCurrency() takes an ICU Currency object which
+    encapsulates the 3-letter code.
+
+4.  The base JDK's NumberFormat.setCurrency() takes a JDK Currency object which
+    encapsulates the 3-letter code.
+
+The functionality of Currency and setCurrency() is more advanced in ICU than in
+the base JDK. When using ICU, setting the currency automatically adjusts the
+number format object appropriately, i.e., it sets not only the currency symbol
+and display name, but also the correct number of fraction digits and the correct
+[rounding mode](rounding-modes.md). This is not the case with the base JDK. See
+the API references for more details.
+
+There is ICU4C sample code at
+[icu/source/samples/numfmt/main.cpp](http://source.icu-project.org/repos/icu/trunk/icu4c/source/samples/numfmt/main.cpp)
```

Comment:
Should the sample point to `https://github.com/unicode-org/icu/blob/master/icu4c/source/samples/numfmt.cpp` instead?

Reply:
Good catch!